src/averageAges.js

```diff
@@ -15,11 +15,17 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const menAverageAges = (arguments.length > 1)
+    ? people
+      .filter((p) => p.sex === 'm' && Math.ceil(p.died / 100) === century)
```

Comment:
```suggestion
const MAN = 'm';
const YEARS_IN_CENTURY = 100;
const isMan = p => p.sex === MAN;

      .filter((p) => isMan(p) && Math.ceil(p.died / YEARS_IN_CENTURY) === century)
```

Reply:
Consider using constants with meaningful names instead of magic strings and numbers and use the DRY principle for your code: use functions for repeating functionality here